wormhole-connect/src/utils/vaa.ts

```diff
@@ -11,6 +15,61 @@ import {
   getCurrentBlock,
   isEvmChain,
 } from './sdk';
+import { repairVaa } from './repairVaa';
+
+type GuardianSetData = {
+  index: number;
+  keys: string[];
+  expiry: number;
+};
+
+const GUARDIAN_SETS: { [key: string]: GuardianSetData } = {
```

Comment:
`key` should be `Network`, right? there's a type for that.

also, we tend to keep network-specific constants like this here:

https://github.com/wormhole-foundation/wormhole-connect/blob/development/wormhole-connect/src/config/mainnet/

for example, built-in tokens, RPCs, etc all live in there. can you move this const to there?